README.md

```diff
@@ -28,6 +28,10 @@ bash alias dcp="docker-compose -f docker-compose-production.yml"
 
 #### Готовим код к работе
 ```bash
+mkdir fidals && cd fidals
+# clone outer dependency repository
```

Comment:
It's not fully outer. It's outer for SE repo, but inner for SE project.
"clone sites shared code" would be more good